vmm/src/config.rs

```diff
@@ -2288,6 +2297,25 @@ impl RestoreConfig {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub struct TpmConfig {
+    pub path: PathBuf,
+}
+
+impl TpmConfig {
+    pub const SYNTAX: &'static str = "TPM device \
+        \"(UNIX Domain Socket from swtpm) path=</path/to/a/file>\"";
```

Comment:
If we only want to support a UNIX socket, then we should be more specific with the parameter name and call it `socket` instead of `path`. We might want to keep `path` if we plan on relying on a device on the host by providing a path like `/dev/tpm`.
@rbradford wdyt?

Reply:
Make sense to have the parameter name as `socket` instead of `path`. I will go ahead and make this change.

Reply:
Socket is good.

Reply:
fixed